Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticTypes.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Physics
+{
+
+    /// <summary>
+    /// Properties of the extent in which a damped
+    /// harmonic oscillator is free to move.
+    /// </summary>
+    [Serializable]
+    public struct ElasticExtentProperties<T>
+    {
+        /// <value>
+        /// Represents the lower bound of the extent,
+        /// specified as the norm of the n-dimensional extent
+        /// </value>
+        [SerializeField]
+        public float MinStretch;
```

Comment:
For fields, we typically recommend [private fields with public property accessors (if needed)](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation).